engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -78,6 +79,7 @@ @interface FlutterViewController () <FlutterBinaryMessenger, UIScrollViewDelegat
 @property(nonatomic, assign) UIStatusBarStyle statusBarStyle;
 @property(nonatomic, assign) BOOL initialized;
 @property(nonatomic, assign) BOOL engineNeedsLaunch;
+@property(nonatomic, assign) BOOL wokenFromNib;
```

Comment:
Pedantry nit: since it's awakeFromNib rather than wakeFromNib, this should be `awokenFromNib`.